.github/workflows/releaseSnapshotPackage.yml

```diff
@@ -73,11 +73,13 @@ jobs:
           ref: "${{ env.tag_name }}"
           inputs: '{"release_id": "${{ steps.create_release.outputs.id }}", "python_package_version": "${{ env.package_version }}"}'
 
-      - name: Download nightly pytorch and torchvision
+      - name: Download nightly pytorch and torchvision wheels
         run: |
-          pip download -r pytorch-requirements.txt -r torchvision-requirements.txt --no-deps --dest deps
+          pip download -r pytorch-requirements.txt -r torchvision-requirements.txt --no-deps --dest deps --python-version 3.8
+          pip download -r pytorch-requirements.txt -r torchvision-requirements.txt --no-deps --dest deps --python-version 3.10
+          pip download -r pytorch-requirements.txt -r torchvision-requirements.txt --no-deps --dest deps --python-version 3.11
```

Comment:
Do we use all these different version somewhere or is this just to be safe?

Reply:
3.8 is used by default on Ubuntu 20.04, which most developers use.
3.10 is used on Ubuntu 22.04, which is required to develop on torch-mlir.
3.11 is there because I will surely forget to add the next version here in a year when we need it :-)